models/classes/resources/ListResourceLookup.php

```diff
@@ -74,7 +74,13 @@ private function searchByString($searchString, $rootClass, $offset, $limit)
         /** @var ResultSet $result */
         $result = $searchService->query($searchString, $rootClass, $offset, $limit);
         $count = $result->getTotalCount();
-        return $this->format($result, $count, $offset, $limit);
+
+        $ids = [];
+        foreach ($result as $item) {
+            $ids[] = $item['id'];
+        }
```

Comment:
Nothing important, but this will be a tad less cumbersome.

```suggestion
        $ids = array_column($result, 'id');
```

Reply:
For array_column(), the expected type is 'array'. but here it is 'oat\tao\model\search\ResultSet' object

Reply:
Fair point.